AutoscoperM/AutoscoperMLib/RadiographGeneration.py

```diff
@@ -11,9 +11,60 @@ def __init__(self) -> None:
         self.vtkCamera = vtk.vtkCamera()
         self.imageSize = [512, 512]
         self.id = -1
+        self.FrustumModel = None
 
+    def __str__(self) -> str:
+        dbgStr = f"Camera {self.id}\n"
+        dbgStr += f"Position: {self.vtkCamera.GetPosition()}\n"
+        dbgStr += f"Focal Point: {self.vtkCamera.GetFocalPoint()}\n"
+        dbgStr += f"View Angle: {self.vtkCamera.GetViewAngle()}\n"
+        dbgStr += f"Clipping Range: {self.vtkCamera.GetClippingRange()}\n"
+        dbgStr += f"View Up: {self.vtkCamera.GetViewUp()}\n"
+        dbgStr += f"Direction of Projection: {self.vtkCamera.GetDirectionOfProjection()}\n"
+        dbgStr += f"Distance: {self.vtkCamera.GetDistance()}\n"
+        dbgStr += f"Image Size: {self.imageSize}\n"
+        dbgStr += f"DID: {self.DID}\n"
+        dbgStr += "~" * 20 + "\n"
+        return dbgStr
 
-def generateNCameras(N: int, bounds: list[int], offset: int = 100, imageSize: tuple[int] = (512, 512)) -> list[Camera]:
+
+def _createFrustumModel(cam: Camera) -> None:
+    model = slicer.mrmlScene.AddNewNodeByClass("vtkMRMLModelNode")
+    model.CreateDefaultDisplayNodes()
+    model.GetDisplayNode().SetColor(1, 0, 0)
+    model.GetDisplayNode().SetOpacity(0.3)
+    model.GetDisplayNode().SetVisibility(True)
+
+    model.SetName(f"cam{cam.id}-frustum")
+
+    cam.FrustumModel = model
+
+    _updateFrustumModel(cam)
+
+
+def _updateFrustumModel(cam: Camera) -> None:
+    if cam.FrustumModel is None:
+        _createFrustumModel(cam)
+        return
+    planesArray = [0] * 24
```

Comment:
Should a constant be introduced instead of hard-coding `24` ?

Reply:
I don't think this is necessary since the size of the array is from VTK, see [vtkCamera.GetFrustumPlanes()](https://vtk.org/doc/nightly/html/classvtkCamera.html#a63ac225662fc7b81628366a7b3b7096c).

I did add a comment describing where this comes from.